ecyp-database.Rmd

```diff
@@ -87,10 +100,10 @@ refresh_db <- function(sheet_url) {
   authenticate()
 
   # Read in sheet
-  ecyp_db_org <- gsheet::gsheet2tbl(sheet_url) %>%
+  ecyp_db_org <- googlesheets4:: read_sheet(sheet_url) %>%
```

Comment:
It doesn't really matter which one we use, but I changed this because I was testing out a thing.